certified-connectors/ShareEffect/apiDefinition.swagger.json

```diff
@@ -171,8 +168,8 @@
                         "parameterReference": "body/termsgroup"
                       }
                     },
-                    "itemValuePath": "key-item-output",
-                    "itemTitlePath": "key-item-output"
+                    "itemValuePath": "",
+                    "itemTitlePath": ""
```

Comment:
@Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"